src/pspm_sf.m

```diff
@@ -199,7 +215,26 @@
       'Please check your results.\n'], ...
       data{1}.header.units);
   end
-  % 3.5 Get marker data
+  % 3.5 Get missing epochs
+  if ~isempty(model.missing{iFile})
+    [~, missing{iFile}] = pspm_get_timing('epochs', ...
+      model.missing{iFile}, 'seconds');
+    % sort missing epochs
+    if size(missing{iFile}, 1) > 0
+      [~, sortindx] = sort(missing{iFile}(:, 1));
+      missing{iFile} = missing{iFile}(sortindx,:);
+      % check for overlap and merge
+      for k = 2:size(missing{iSn}, 1)
+        if missing{iFile}(k, 1) <= missing{iFile}(k - 1, 2)
+          missing{iFile}(k, 1) =  missing{iFile}(k - 1, 1);
+          missing{iFile}(k - 1, :) = [];
+        end
+      end
+    end
```

Comment:
In the future, this code (and the same for pspm_dcm and pspm_data_editor) could be included in pspm_get_timing (optional). Could you open a code improvement issue for this when you merge here?

Reply:
I agree. I have referred this comment and code in the new issue (improvement).